frontend/apps/reader/modules/readercoptlistener.lua

```diff
@@ -18,10 +26,179 @@ function ReaderCoptListener:onReadSettings(config)
 
     local status_line = config:readSetting("copt_status_line") or G_reader_settings:readSetting("copt_status_line") or 1
     self.ui:handleEvent(Event:new("SetStatusLine", status_line, true))
+
+    self.title = G_reader_settings:readSetting("cre_header_title") or 1
+    self.clock = G_reader_settings:readSetting("cre_header_clock") or 1
+    self.page_number = G_reader_settings:readSetting("cre_header_page_number") or 1
+    self.page_count = G_reader_settings:readSetting("cre_header_page_count") or 1
+    self.reading_percent = G_reader_settings:readSetting("cre_header_reading_percent") or 0
+    self.battery = G_reader_settings:readSetting("cre_header_battery") or 1
+    self.battery_percent = G_reader_settings:readSetting("cre_header_battery_percent") or 0
+    self.chapter_marks = G_reader_settings:readSetting("cre_header_chapter_marks") or 1
+
+    self.ui.document._document:setIntProperty("window.status.title", self.title)
+    self.ui.document._document:setIntProperty("window.status.clock", self.clock)
+    self.ui.document._document:setIntProperty("window.status.pos.page.number", self.page_number)
+    self.ui.document._document:setIntProperty("window.status.pos.page.count", self.page_count)
+    self.ui.document._document:setIntProperty("crengine.page.header.chapter.marks", self.chapter_marks)
+    self.ui.document._document:setIntProperty("window.status.battery", self.battery)
+    self.ui.document._document:setIntProperty("window.status.battery.percent", self.battery_percent)
+    self.ui.document._document:setIntProperty("window.status.pos.percent", self.reading_percent)
 end
 
 function ReaderCoptListener:onSetFontSize(font_size)
     self.document.configurable.font_size = font_size
 end
```

Comment:
Hint for @yparitcher  : looks like this event handler is just there to update the bottom menu state (the main handler is in ReaderFont).
So, at worst, add such stupid handlers in readercoptlistener.lua and readerkoptlistener.lua ?
(Althought I'm not sure why there is this one, and only this one :)

Reply:
this is basically re implementing `ConfigDialog` stuff in other places. If we go this route we should probably remove the configurable stuff from `ConfigDialog` and have it done once, leaving the event as the source of truth.